src/newrelic.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['tta-smart-hub-development'],
+  /**
+   * This setting controls distributed tracing.
+   * Distributed tracing lets you see the path that a request takes through your
+   * distributed system. Enabling distributed tracing changes the behavior of some
+   * New Relic features, so carefully consult the transition guide before you enable
+   * this feature: https://docs.newrelic.com/docs/transition-guide-distributed-tracing
+   * Default is true.
+   */
+  distributed_tracing: {
+    /**
+     * Enables/disables distributed tracing.
+     *
+     * @env NEW_RELIC_DISTRIBUTED_TRACING_ENABLED
+     */
+    enabled: true,
+  },
+  logging: {
+    /**
+     * Level at which to log. 'trace' is most useful to New Relic when diagnosing
+     * issues with the agent, 'info' and higher will impose the least overhead on
+     * production applications.
+     */
+    level: 'info',
+    filepath: 'stdout',
+  },
+  /**
+   * When true, all request headers except for those listed in attributes.exclude
+   * will be captured for all traces, unless otherwise specified in a destination's
+   * attributes include/exclude lists.
+   */
+  allow_all_headers: true,

```

Comment:
I'm think it would be safer here to exclude all headers and then set an inclusions list. (It seems like that's an option from the comment above.) I can see us accidentally logging PII if an engineer doesn't know or forgets that the default is to log all headers.

Reply:
Do you have an initial list of headers we should include?

Reply:
Here's my initial list. I'll commit it if you agree it's a good list:

```
    include: [
      'request.headers.accept',
      'request.headers.host',
      'request.headers.referer',
      'request.headers.userAgent',
      'response.headers.contentType',
      'response.headers.date',
    ],
```

Reply:
This looks like a good list to me! Thanks for brainstorming that.